.pre-commit-config.yaml

```
@@ -16,6 +16,13 @@ repos:
         args:
           - "--profile"
           - "black"
+  - repo: https://github.com/doublify/pre-commit-rust
+    rev: master
```

pre-commit throws up a warning whenever it runs because we aren't using a tagged version here - would you mind switching to

```suggestion
    rev: v1.0
```